Makefile

```diff
@@ -5,3 +5,18 @@ plan-infra:
 
 deploy-infra:
 	$(MAKE) -C infra apply-all
+
+requirements.txt: %.txt : %.txt.in
+	[ ! -e .requirements-env ] || exit 1
```

Comment:
Nit: this would be clearer as a call to `test` instead of using bracket syntax

Reply:
@natanlao Would this require replacing `[ ... ] || exit 1` with `test -e .requirements-env || exit 1`?

I do like the bracket syntax for its conciseness, but agree `test` is more readable. If you make a suggested change I'll accept it